packages/jest-cli/src/TestRunner.js

```diff
@@ -218,7 +218,8 @@ class TestRunner {
       );
       aggregatedResults.snapshot.filesRemoved += status.filesRemoved;
       aggregatedResults.snapshot.didUpdate = config.updateSnapshot;
-      aggregatedResults.snapshot.failure = !!(!aggregatedResults.snapshot.didUpdate &&
+      aggregatedResults.snapshot.failure =
```

Comment:
I want to do this at some point: https://github.com/prettier/prettier/issues/774